app/src/main/java/kr/ac/konkuk/gdsc/plantory/presentation/plant/DetailAdapter.kt

```diff
@@ -78,7 +80,7 @@ class DetailAdapter(
         val dateFormat = SimpleDateFormat("yyyy/MM/dd", Locale.getDefault())
         val formattedCurrentDate = dateFormat.format(currentDate)
 
-        val plantDailyRecordsForDate = dummyinfo.filter {
+        val plantDailyRecordsForDate = plantHistories.filter {
```

Comment:
```suggestion
        val plantDailyRecordsForDate = currentList.filter {
```

이렇게 현재 adapter내의 list를 다시 가져올 수 있습니다.